src/test/java/uk/gov/hmcts/reform/dg/docassembly/service/DocmosisApiClientTest.java

```diff
@@ -0,0 +1,80 @@
+package uk.gov.hmcts.reform.dg.docassembly.service;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import okhttp3.Call;
+import okhttp3.OkHttpClient;
+import org.jetbrains.annotations.NotNull;
```

Comment:
import org.jetbrains.annotations.NotNull;
Can we use the one provided by apache commons library? To be consistent across the API. Let us not tie ourselves to particular IDE library.